### DISCUSSION TITLE
Markdown s3 upload not working (404 error on /nova-api/campaigns/field-attachment/copy)

### DISCUSSION THREAD

**Discussion by HeadStudios** [laravel/nova-issues#5572] [Q&A] [answered]
- Laravel Version: 9
- Nova Version: 4
- PHP Version: 8.2
- Database Driver & Version: MySQL
- Operating System and Version: MacOS Big Sur
- Browser type and version: Version 113.0.5672.126 (Official Build) (arm64)
- Reproduction Repository: https://github.com/HeadStudios/novaissue-5456-nonova

### Description:

I am unable to upload files through s3 to Markdown field (withFiles('s3') option) - getting 404 error even though the URL itself exists:
https://share.getcloudapp.com/o0uJN7O9

https://share.getcloudapp.com/2NupJPo5

Video for context: https://p147.p4.n0.cdn.getcloudapp.com/items/rRuzwWg9/9fdd6dd8-1261-454c-8c90-66a1b5099633.mp4?v=a2afebc8135a83b8c7c3bd48a97b05d6

### Detailed steps to reproduce the issue on a fresh Nova installation:

Setup Campaign.php model as so: https://github.com/HeadStudios/novaissue-5456-nonova/blob/main/app/Models/Campaign.php

**Comment by crynobone** [comment]
404 here can be a result where Nova is unable to identify that `copy` field attribute exists on the Resource. This could happen when 3rd party package doesn't compile fields inside the custom panel correctly.

**Reply by crynobone** [reply, marked as answer]
https://github.com/Formfeed-UK/nova-dependable-panel/blob/main/src/DependablePanel.php


`DependablePanel` extends `Field` and not `Panel`. As a result, Laravel Nova can't verify that the field exists via attribute name. This mean Nova doesn't know the correct Markdown field to be referred to and returns `404`. You can resolve this using a normal Panel or find a way around it.

**Reply by HeadStudios** [reply]
Thank you! This has fixed my issue.
